Library/ENV/scm/git

```diff
@@ -10,8 +10,8 @@ fi
 exec "$HOMEBREW_RUBY_PATH" -x "$0" "$@"
 #!/usr/bin/env ruby -W0
 
-# This script because we support $GIT, $HOMEBREW_SVN, etc. and Xcode-only
-# configurations. Order is careful to be what the user would want.
+# This script because we support $GIT, $HOMEBREW_SVN, etc., Xcode-only and
```

Comment:
The plain `GIT` also creates a kind of asymmetry when looking at the Subversion part of the script. Only supporting `HOMEBREW_GIT` and `HOMEBREW_SVN` would make things a bit more uniform. Thus I'm somewhat in favor of no longer supporting plain `GIT`.